x-pack/plugins/security_solution_serverless/kibana.jsonc

```diff
@@ -19,7 +19,8 @@
       "securitySolution",
       "serverless",
       "taskManager",
-      "cloud"
+      "cloud",
+      "fleet"
```

Comment:
Q: what is this change for?

Reply:
this is to access `PackagePolicyClient` from fleet so that we can list/update endpoint package policies